app/styles/_shame.scss

```diff
@@ -31,5 +41,24 @@ table.dataTable tbody th {
 }
 
 .yasgui .autocompleteWrapper {
-  display: none !important;
+  @extend .au-u-hidden;
+}
+
+.au-c-data-table__table td:first-child {
+  @extend .au-u-word-break;
+}
+
+// Fix min-height to show prev/next buttons on all screens
+
+.au-c-main-container {
+  min-height: calc(100vh - 15rem);
+}
+
+/* Styles for status-filter */
+
+.filter-select {
```

Comment:
Nitpick: This is a custom component, so it shouldn't be in `_shame.scss` since it will never be moved to Appuniversum.